src/lando/api/views.py

```diff
@@ -104,3 +110,45 @@ class Form(forms.Form):
             )
 
         return JsonResponse({"errors": dict(form.errors)}, status=400)
+
+
+class CommitMapBaseView(View):
+    """CommitMap base view to be extended for bidirectional git - hg mapping."""
+
+    scm = None
+
+    def get(
+        self, request: WSGIRequest, git_repo_name: str, commit_hash: str
```

Comment:
nit: depending on the file, we have type annotations of `request` as `django.http.HttpRequest` or this `WSGIRequest`. We should probably try to choose one for consistency, though I'm not sure which one, as I don't know what the difference is off the bat, do you know?

Reply:
As a nit, I'd say we should stick to the HttpRequest superclass, unless we are using things that are specific to WSGI.

Reply:
The entire app is wired to be a wsgi one. So in effect, things are quite specific to wsgi even though in theory we could change things in the future to support asgi. That is to say, this argument will never be anything other than WSGIRequest unless we explicitly change the structure of this project.

I am not too picky here and think both work, but I do think that WSGIRequest is a more appropriate type especially since we never actually call any of these methods directly, but they are handled by the dispatcher.

Happy to do either, so we can make this more consistent but let's do that in a separate PR.